packages/ui/src/Box.tsx

```diff
@@ -43,8 +44,33 @@ const ALIGN_SELF = {
 
 const BORDER_WIDTH = 1;
 
-export class Box extends React.Component<BoxProps, {}> {
-  BOX_STYLE_MAP: {
+// eslint-disable-next-line react/display-name
+export const Box = React.forwardRef((props: BoxProps, ref) => {
+  useImperativeHandle(ref, () => ({
+    scrollToEnd: () => {
+      if (scrollRef && scrollRef.current) {
+        // HACK HACK HACK...but it works. Probably need to do some onContentSizeChange or onLayout to
+        // avoid this, but it works well enough.
+        setTimeout(() => {
+          scrollRef && scrollRef.current && (scrollRef.current as any).scrollToEnd();
+        }, 50);
+      }
+    },
+
+    scrollTo: (y: number) => {
+      if (scrollRef && scrollRef.current) {
+        // HACK HACK HACK...but it works. Probably need to do some onContentSizeChange or onLayout to
+        // avoid this, but it works well enough.
+        setTimeout(() => {
+          scrollRef && scrollRef.current && (scrollRef.current as any).scrollTo({y});
+        }, 50);
+      }
+    },
+  }));
+
+  const [_, setHovered] = React.useState(false);
```

Comment:
Learning moment for me. Not sure what this does.. how do we read this state?

Reply:
Ah it doesn't actually do anything. I put it in thinking Box would do some hover stuff itself but it wound up calling the props instead. I'll remove.